packages/@aws-cdk/aws-cloudwatch/README.md

```diff
@@ -274,6 +274,18 @@ dashboard.addWidgets(new TextWidget({
 }));
 ```
 
+### Query results widget
+
+A query results widget shows the results of a query from Logs Insights:
+
+```ts
+dashboard.addWidgets(new QueryWidget({
+  logGroup: LogGroup.fromLogGroupName(stack, 'MyLogGroup', 'my-log-group'),
+  query: `fields @message
```

Comment:
One more thing, can you rename `query` to `queryString` ? I feel it will help us in the future if we ever want to provide query builder objects, which can then be used as `query: Query.fromFields(...)` or whatever.

Reply:
Done!